### PR TITLE
Add load privileges

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates.

You can view the updated docs at https://neo4j-docs-operations-1066.surge.sh

File: modules/ROOT/pages/authentication-authorization/built-in-roles.adoc [hunk 1]
```diff
@@ -45,11 +45,11 @@ In other words, the `editor` role cannot add to the schema but can only make cha
 <<access-control-built-in-roles-admin, `admin`>>::
 * Access to all databases.
 * Traverse, read, and write on the data graph.
-* Allows loading data.
+* Load data.
```
Comment (Hunterness) [inline modules/ROOT/pages/authentication-authorization/built-in-roles.adoc:48]: should you do these updates for `PUBLIC` as well and not just `admin`?

File: modules/ROOT/pages/authentication-authorization/built-in-roles.adoc [hunk 2]
```diff
@@ -45,11 +45,11 @@ In other words, the `editor` role cannot add to the schema but can only make cha
...
 * Create/drop/show indexes and constraints along with any other future schema constructs.
-* Allows executing procedures using boosted privileges.
-* Allows executing admin procedures.
-* Allows executing user-defined functions using boosted privileges.
+* Execute procedures using boosted privileges.
+* Execute admin procedures.
+* Execute user-defined functions using boosted privileges.
 * View/terminate queries.
 * Manage databases, users, roles, and privileges.
```
Comment (Hunterness) [inline modules/ROOT/pages/authentication-authorization/built-in-roles.adoc:55]: The `load data` row is duplicated in the table below

**Review by Hunterness — APPROVED** [review]
Just two small things I found

File: modules/ROOT/pages/authentication-authorization/built-in-roles.adoc (hunk 2 above)
Comment (renetapopova) [reply modules/ROOT/pages/authentication-authorization/built-in-roles.adoc:55]: Thanks!

File: modules/ROOT/pages/authentication-authorization/built-in-roles.adoc (hunk 1 above)
Comment (renetapopova) [reply modules/ROOT/pages/authentication-authorization/built-in-roles.adoc:48]: Of course, thanks!

**Pull request merged**